src/compose/ComposeBox.js

```diff
@@ -181,15 +183,12 @@ class ComposeBoxInner extends PureComponent<Props, State> {
     }));
   };
 
-  getCanSelectTopic = () => {
+  canShowTopicInputBox = () => {
     const { isEditing, narrow } = this.props;
     if (isEditing) {
       return isStreamOrTopicNarrow(narrow);
     }
-    if (!isStreamNarrow(narrow)) {
-      return false;
-    }
-    return this.state.isFocused;
+    return isStreamNarrow(narrow);
```

Comment:
I think the old behaviour is better, so I changed the callsite to take that into account. This isn't super clean (I think a better approach would be to change `canShowTopicInputBox` to return `"never" | "whenFocused" | "always"`, but I don't feel like making that change right now, since I think we may still want to change this code in other ways.